mix.exs

```diff
@@ -65,7 +65,7 @@ defmodule Membrane.Element.FLACParser.MixProject do
     [
       {:membrane_core, "~> 0.5.0"},
       {:membrane_caps_audio_flac, "~> 0.1.1"},
-      {:crc, "~> 0.9.2"},
+      {:crc, "~> 0.10"},
```

Comment:
Specify path number when major is 0, see https://semver.org/ and https://hexdocs.pm/elixir/Version.html
```suggestion
      {:crc, "~> 0.10.0"},
```